content/docs/command-reference/move.md

```diff
@@ -69,6 +71,46 @@ outs:
     path: other.csv
 ```
 
+### Manually renaming stage outputs
+
+In this example, let's rename stage output `keras.h5` to `model.h5`:
+
+```dvc
+$ cat dvc.yaml
+stages:
```

Comment:
Better to use a YAML code block, no need for `$ cat` here. May need to change the context a little too.